File: src/main/java/com/train/Tester.java
package com.train;

import java.util.Scanner;

public class Tester {
    private static int ticket;
    private static int roundticket;
    public static void main(String[] args) {

        Scanner scanner = new Scanner(System.in);
        while(ticket == 0){
            System.out.println("Please enter number of tickets: ");
            ticket = scanner.nextInt();
            if (ticket >= 0){
                break;
            }
        }
        while(roundticket == 0){
            System.out.println("How many round-trip tickets: ");
            roundticket = scanner.nextInt();
            if(roundticket > ticket){
                roundticket = 0;
            }
        }
        Ticket money = new Ticket(ticket, roundticket);
        money.print();
    }
}

File: src/main/kotlin/com/train/Tester.kt
package com.train

import java.util.*

fun main(args: Array<String>) {
    val scanner = Scanner(System.`in`)
    var ticket :Int
    var roundticket :Int
    while(true){
        print("Please enter number of tickets: ")
        ticket = scanner.nextInt()
        if(ticket > 0){
            break
        }
    }
    while(true){
        print("How many round-trip tickets: ")
        roundticket = scanner.nextInt()
        if (roundticket <= ticket){
            break
        }
    }
    val money =Ticket2(ticket,roundticket)
    money.print()
}

class Ticket2(var ticket : Int,var roundticket : Int){
    fun print(){
        val Total :Int = ((ticket-roundticket)*1000 + roundticket*2000*0.9).toInt()
        println("Total tickets:" + ticket)
        println("Round-trip:" + roundticket)
        println("Total:" + Total)
    }
}